exercises/diffie-hellman/src/test/java/DiffieHellmanTest.java

```diff
@@ -0,0 +1,95 @@
+import org.junit.Before;
+import org.junit.Ignore;
+import org.junit.Test;
+
+import static org.junit.Assert.assertTrue;
+import static org.junit.Assert.assertEquals;
+
+import java.math.BigInteger;
+import java.util.*;
+
+public class DiffieHellmanTest {
+
+    private DiffieHellman diffieHellman;
+
+    @Before
+    public void setUp() {
+        diffieHellman = new DiffieHellman();
+    }
+
+    @Test
+    public void testPrivateKeyInRange() {
+        BigInteger primeP = BigInteger.valueOf(23);
```

Comment:
Why the name `primeP`? The name should describe what the variable is for as accurately as possible. If this variable is only meant to be some prime then just `prime` might be a more descriptive name? 🙂

Reply:
True 🙂